software/controller/lib/core/sensors.h

```diff
@@ -81,12 +84,14 @@ class Sensors {
     PATIENT_PRESSURE,
     INFLOW_PRESSURE_DIFF,
     OUTFLOW_PRESSURE_DIFF,
+    FIO2,
```

Comment:
Let's take the opportunity to fix this to follow google style, and also, this should likely be an `enum class` rather than old style `enum`.